spec/controllers/course/controller_spec.rb

```diff
@@ -7,4 +7,11 @@
       expect(controller.sidebar).to eq([])
     end
   end
+
+  describe '#settings' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.